content/fr/docs/concepts/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Concepts
+main_menu: true
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+
+La section Concepts vous aide à mieux comprendre les composants du système Kubernetes et les abstractions que Kubernetes utilise pour représenter votre cluster.
+Elle vous aide également à mieux comprendre le fonctionnement de Kubernetes en général.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Vue d'ensemble
+
+Pour utiliser Kubernetes, vous utilisez *les objets de l'API Kubernetes* pour décrire *l'état souhaité* de votre cluster: quelles applications ou autres processus que vous souhaitez exécuter, quelles images de conteneur elles utilisent, le nombre de réplicas, les ressources réseau et disque que vous mettez à disposition, et plus encore.
+Vous définissez l'état souhaité en créant des objets à l'aide de l'API Kubernetes, généralement via l'interface de ligne de commande, `kubectl`.
+Vous pouvez également utiliser l'API Kubernetes directement pour interagir avec le cluster et définir ou modifier l'état souhaité.
+
+Une fois que vous avez défini l'état souhaité, le *Plan de contrôle Kubernetes* permet de faire en sorte que l'état actuel du cluster corresponde à l'état souhaité.
+Pour ce faire, Kubernetes effectue automatiquement diverses tâches, telles que le démarrage ou le redémarrage de conteneurs, la mise à l'échelle du nombre de réplicas d'une application donnée, etc.
+Le plan de contrôle Kubernetes comprend un ensemble de processus en cours d'exécution sur votre cluster:
+
+* Le **maître Kubernetes** qui est un ensemble de trois processus qui s'exécutent sur un seul nœud de votre cluster, désigné comme nœud maître. Ces processus sont: [kube-apiserver](/docs/admin/kube-apiserver/), [kube-controller-manager](/docs/admin/kube-controller-manager/) et [kube-scheduler](/docs/admin/kube-scheduler/).
```

Comment:
i personnaly use Master, it's ok tu use maître if we all agree, cause it's better to use same words